p2p/README.md

```diff
@@ -1 +1,20 @@
-# Stronghold P2p
+# Stronghold Peer-to-Peer Communication
+
+The Stronghold-p2p library enables end-to-end encrypted communication between peers in different processes, devices and networks.
+The basis for its functionality is the [libp2p](https://libp2p.io/) framework, which is a system of protocols, specifications and libraries that enable the development of peer-to-peer network applications.
+
+The crate is build separately of Stronghold, and may be used independently. It allows the user to transmit generic 1:1 Request-Response messages between two peers, with an additional firewall to prevent unauthorized access. In case that a peer may not be dialed directly, it optionally supports the usage of relay peer that blindly relays the traffic between two peers.
```

Comment:
```suggestion
You can build the Stronghold-p2p crate separately from Stronghold, as well as use it independently. It allows users to transmit generic 1:1 Request-Response messages between two peers, with an additional firewall that prevents unauthorized access. In case that a peer may not be dialed directly, it supports the usage of a relay peer that blindly relays the traffic between two peers.
```